internal/vulnerability/runner.go

```diff
@@ -48,17 +49,17 @@ func NewVulnerabilityRunner(log *logp.Logger) (VulnerabilityRunner, error) {
 			Quiet:   false,
 			Debug:   true,
 		},
-		VulnerabilityOptions: flag.VulnerabilityOptions{
-			VulnType: []string{trivy_types.VulnTypeOS, trivy_types.VulnTypeLibrary},
+		PackageOptions: flag.PackageOptions{
+			PkgTypes: []string{trivy_types.PkgTypeOS, trivy_types.PkgTypeLibrary},
 		},
 		ScanOptions: flag.ScanOptions{
 			Scanners: []trivy_types.Scanner{trivy_types.VulnerabilityScanner},
 			RekorURL: "https://rekor.sigstore.dev",
 		},
 		DBOptions: flag.DBOptions{
 			NoProgress:       true,
-			DBRepository:     "ghcr.io/aquasecurity/trivy-db",
-			JavaDBRepository: "ghcr.io/aquasecurity/trivy-java-db",
+			DBRepository:     name.MustParseReference("ghcr.io/aquasecurity/trivy-db"),
+			JavaDBRepository: name.MustParseReference("ghcr.io/aquasecurity/trivy-java-db"),
```

Comment:
Fix breaking change introduced by adoption of name.Reference on DBRepositories (https://github.com/aquasecurity/trivy/pull/6410)